core/templates/domain/suggestion/suggestion.model.ts

```diff
@@ -74,30 +70,25 @@ export class Suggestion {
     this.lastUpdatedMsecs = lastUpdatedMsecs;
   }
 
+  static createFromBackendDict(
+    suggestionBackendDict: SuggestionBackendDict): Suggestion {
+  let threadId = this.suggestionsService.getThreadIdFromSuggestionBackendDict(
+    suggestionBackendDict);
+  return new Suggestion(
+    suggestionBackendDict.suggestion_type,
+    suggestionBackendDict.suggestion_id,
+    threadId,
+    suggestionBackendDict.target_type,
+    suggestionBackendDict.target_id,
+    suggestionBackendDict.status,
+    suggestionBackendDict.author_name,
+    suggestionBackendDict.change.state_name,
+    suggestionBackendDict.change.new_value,
+    suggestionBackendDict.change.old_value,
+    suggestionBackendDict.last_updated_msecs);
```

Comment:
Please fix indentation of all of these lines (+2 spaces for 74-88)

Reply:
Thanks , done!